modules/pipeline/services/pipelinesvc/rerun.go

```diff
@@ -36,6 +36,11 @@ func (s *PipelineSvc) Rerun(req *apistructs.PipelineRerunRequest) (*spec.Pipelin
 		originCron = cron
 	}
 
+	if origin.Labels == nil {
```

Comment:
Labels will not be nil. `s.dbClient.GetPipelineCron` guranteed it.

Reply:
retrying the entire pipeline does not necessarily mean retrying a cron pipeline. It is understandable to add a layer of judgment